src/components/NewMessageModal.vue

```diff
@@ -11,6 +11,7 @@
 			:subject="composerData.subject"
 			:attachments-data="composerData.attachments"
 			:body="composerData.body"
+			:editor-body="composerData.editorBody"
```

Comment:
what happens if `editorBody` is null? Every installation with existing outbox messages will run into this.

Reply:
https://github.com/nextcloud/mail/pull/6372#issuecomment-1143325891

Reply:
i would say this is solved